src/pages/Cart/index.js

```diff
@@ -64,24 +76,54 @@ function ProductCart() {
 }
 
 function Carting({ product }) {
+  const [quantity, setQuantity] = useState(1);
+  function addQuantity() {
+    setQuantity(quantity + 1);
+  }
+
+  const store = useCartStore();
+
+  function handleDeleteProduct(product) {
+    store.removeCart(product);
+  }
+
+  function reduceQuantity() {
+    if (quantity > 1) setQuantity(quantity - 1);
+  }
+  const price = +product.price;
+
+  const subTotal = price * quantity;
+
   return (
     <li className="cart-list">
       <div className="cart-item">
         <img src={product.image} alt={product.name} width="70" />
         <div className="cart-details">
-          <p className="cart-button">X Remove</p>
+          <p className="cart-brand">{product.name}</p>
+          <p className="cart-brand">{product.company}</p>
+          <br />
+          <button
+            className="cart-button"
+            onClick={() => handleDeleteProduct(product)}
+          >
+            X Remove
+          </button>
         </div>
       </div>
 
       <div className="numbers">
         <div className="quantity">
-          <p> - </p>
-          <p>1</p>
-          <p> + </p>
+          <button className="cart-button" onClick={reduceQuantity}>
+            <p>-</p>
+          </button>
+          <p>{quantity}</p>
+          <button className="cart-button" onClick={addQuantity}>
+            <p>+</p>
+          </button>
         </div>
 
-        <p className="cart-price">{product.price}</p>
-        <p className="cart-totals">{product.totalPrice}</p>
+        <p className="cart-price">${product.price}</p>
+        <p className="cart-totals"> ${subTotal}.00</p>
```

Comment:
Please remove the `.00` we're hardcoding at the end. Try not to manipulate numbers that way. What if it happens that the subtotal value is something like `23.54`? Then you'd be displaying `23.54.00`, which isn't right

If you want it consistently formatted to 2 decimal places, you can checkout the [ToFixed method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed). It generally works

Reply:
It was just pure laziness sir